src/ReaderScreen/hooks/usePrevious.js

```diff
@@ -0,0 +1,13 @@
+import { useRef, useEffect } from "react";
+
+function usePrevious(value) {
```

Comment:
I am not sure if function keyword is consistent, or the arrow functions can you please check. 

`const usePrevious(value) = () { ... }`

Would suggest switching to arrow body functions, if its a mix of both.

Reply:
fixed